### ISSUE TITLE
Import standard assets to project when a module is installed

### ISSUE THREAD

**Issue by FejZa** [realitycollective/realitytoolkit.dev#72]
# Reality Collective - Reality Toolkit Feature Request

All standard assets, such as materials, must be imported into the projects assets folder and not live in the package itself. We need to move them to the modules respective RealityToolkit.Generated/XYZ/Assets folder. This is specifically important so that default materials can be converted to URP / HDRP as needed.

**Comment by SimonDarksideJ** [comment]
Not an issue, we already have a packaging system there, so likely it will need expanding regarding materials and shaders.  Textures should still be fine in the package

Does raise a question about whether it should be "automatically" imported when a package is added, or still a manual task.  Currently the user is required to install the package from the Editor menu.

**Comment by FejZa** [comment]
There is no point in keeping textures in the package. Why? Everything that the user might want to alter should be imported into assets. Especially since we already have a system in place, that does exactly that (PackageInstaller)

**Comment by FejZa** [comment]
Is it really manual? If I remember correctly users get prompted to install after adding a package

**Comment by SimonDarksideJ** [comment]
> Is it really manual? If I remember correctly users get prompted to install after adding a package

Yes, you get the option when the package is installed but if you install multiple at the same time it sometimes does not and if you haven't configured a scene yet, it can get in the way

> Why? Everything that the user might want to alter should be imported into assets.

No sense in duplicating textures as they are read-only anyway, they can use the ones in the package or replace them with their own in their project.  If we ever did replace a texture for whatever reason, there would then be a conflict.
Unavoidable with shaders/materials as they can be updated during edit/runtime.
That's been my experience, especially dealing with the MRTK as materials get updated by their scripts and then overwritten when the editor does a reload and resets from cache.

**Comment by FejZa** [comment]
Got it. Definitely something to look at but not urgent at this point IMO.

**Comment by SimonDarksideJ** [comment]
> Got it. Definitely something to look at but not urgent at this point IMO.

Agreed, we just need to expand the scope of what is copied to the project for now.
